loader/loader.c

```diff
@@ -3654,7 +3654,7 @@ VkResult loader_icd_scan(const struct loader_instance *inst, struct loader_icd_t
                                icd_details[i].full_library_path);
                     break;
                 case LOADER_LAYER_LIB_ERROR_WRONG_BIT_TYPE: {
-                    loader_log(inst, VULKAN_LOADER_DRIVER_BIT, 0, "Requested layer %s was wrong bit-type. Ignoring this JSON",
+                    loader_log(inst, VULKAN_LOADER_DRIVER_BIT, 0, "Requested ICD %s was wrong bit-type. Ignoring this JSON",
```

Comment:
Should we say "driver" because of drivers such as lavapipe, etc?

Reply:
I did ICD because the prior message refers to it as ICD's
```
DEBUG | DRIVER:   Searching for ICD drivers named /usr/lib/i386-linux-gnu/libvulkan_intel_hasvk.so
DRIVER:           Requested layer /usr/lib/i386-linux-gnu/libvulkan_intel_hasvk.so was wrong bit-type. Ignoring this JSON
```